zerver/webhooks/slack/view.py

```diff
@@ -8,7 +8,7 @@
 from zerver.decorator import REQ, has_request_variables, api_key_only_webhook_view
 from zerver.models import UserProfile
 
-ZULIP_MESSAGE_TEMPLATE = "**{message_sender}**: `{text}`"
+ZULIP_MESSAGE_TEMPLATE = u"**{message_sender}**: `{text}`"
```

Comment:
I feel like it'd be good to have a system for avoiding this sort of thing in our webhooks in general, but I suppose maybe moving to Python 3 will make the problem go away anyway.